### PR TITLE
add info on Assistant's custom rules editor

### PR DESCRIPTION
**Pull request by khorne3**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team

### CODE REVIEW

File: docs/semgrep-assistant/overview.md [hunk 1]
```diff
@@ -86,6 +86,14 @@ This information is also available in Semgrep Cloud Platform on the **Dashboard*
 ![Semgrep Assistant's priority inbox email and Dashboard view](/img/priority-inbox.png#md-width)
     *Figure*. Semgrep Assistant's priority inbox Dashboard view.
 
+### Custom rules editor
+
+Semgrep Assistant can help you write custom rules to find patterns and vulnerabilities specific to your codebase. The only information you need to provide is the following:
```
Comment (s-santillan) [inline docs/semgrep-assistant/overview.md:91]:
```suggestion
Semgrep Assistant can help you write custom rules to find patterns and vulnerabilities specific to your codebase. The only information you need to provide are the following:
```
The list contains 3 items

File: docs/semgrep-assistant/overview.md [hunk 2]
```diff
@@ -86,6 +86,14 @@ This information is also available in Semgrep Cloud Platform on the **Dashboard*
...
+
+1. An example of bad code
+2. An example of good code
+3. A prompt describing what you want the rule to do in human language
```
Comment (s-santillan) [inline docs/semgrep-assistant/overview.md:95]:
```suggestion
3. A prompt describing what you want the rule to do in human language
```
Do you mean natural language or English?

File: docs/semgrep-assistant/getting-started.md [hunk 3]
```diff
@@ -148,3 +148,22 @@ There is a cap of 250 Assistant runs per month using the **Analyze** button. Ass
...
+3. Click the **plus** button, and under **Generate with AI**, click **...with Semgrep Assistant**.
+   ![The plus button to open up the custom rules editor](/img/assistant-launch-rules-editor.png#md-width)
+4. In the **Generate rule with Semgrep Assistant** pop-up window:
+   1. Select the language of your codebase.
+   2. Provide a prompt describing what you want the rule to do in human-readable language.
+   3. Provide an example of bad code
+   4. Provide an example of good code
+   ![Custom rule sample plus test window](/img/assistant-write-custom-rule.png#md-width)
+5. Click **Generate** to proceed. You'll be redirected to a screen where you can view and copy your rule and test it against the sample bad code snippet you provided.
+   ![Pop-up window to provide custom rule parameters](/img/assistant-view-rule.png#md-width)
```
Comment (s-santillan) [inline docs/semgrep-assistant/getting-started.md:169]:
```suggestion
   ![Dialog box for custom rule parameters](/img/assistant-view-rule.png#md-width)
```
Prefer dialog or dialog box: https://developers.google.com/style/ui-elements#term-windows

File: docs/semgrep-assistant/getting-started.md [hunk 4]
```diff
@@ -148,3 +148,22 @@ There is a cap of 250 Assistant runs per month using the **Analyze** button. Ass
...
 You can [view all of Semgrep Assistant's recommendations](/semgrep-code/findings/#filter-findings) by going to Semgrep Cloud Platform's **Findings** page and filtering by **Recommendation** or **Component**.
+
+## Write custom rules (beta)
+
+Semgrep Assistant can help you write custom rules to find issues specific to your codebase.
+
+To do so:
+
+1. Sign in to Semgrep Cloud Platform.
+2. Navigate to **Rules** > **Editor**.
```
Comment (s-santillan) [inline docs/semgrep-assistant/getting-started.md:159]: We're allowed to **bold** angle brackets : https://developers.google.com/style/procedures

**Review by s-santillan — APPROVED** [review]

File: docs/semgrep-assistant/overview.md [hunk 5]
```diff
@@ -86,6 +86,14 @@ This information is also available in Semgrep Cloud Platform on the **Dashboard*
 ![Semgrep Assistant's priority inbox email and Dashboard view](/img/priority-inbox.png#md-width)
     *Figure*. Semgrep Assistant's priority inbox Dashboard view.
 
+### Custom rules editor
+
+Semgrep Assistant can help you write custom rules to find patterns and vulnerabilities specific to your codebase. The only information you need to provide are the following:
```
Comment (Sly1029) [inline docs/semgrep-assistant/overview.md:91]: The good code and bad code are not necessary to produce an output. They are optional fields that will be used for additional tests and to provide context to the llm

**Pull request merged**